resource/reapi/bindings/c/reapi_cli.h

```diff
@@ -69,7 +70,7 @@ int reapi_cli_initialize (reapi_cli_ctx_t *ctx, const char *rgraph,
  *                   the match operation.
  *  \return          0 on success; -1 on error.
  */
-int reapi_cli_match (reapi_cli_ctx_t *ctx, const char *match_op,
+int reapi_cli_match (reapi_cli_ctx_t *ctx, enum match_op_t match_op,
```

Comment:
No need for `enum` type in function signature.
```suggestion
int reapi_cli_match (reapi_cli_ctx_t *ctx, match_op_t match_op,
```